JWT/jwt.js

```diff
@@ -0,0 +1,15 @@
+var jwtUtil = require('./jwtUtil');
+
+function* jwtChecker(next) {
+  if (this.path != "/shared/auth") {
+    var token = this.headers.token;
```

Comment:
this.headers.Authorization;

var split = token.split('Bearer ');
if (split.length == 2) {
  var id = jwtUtil.decode(split[1]);
} else {
  // error
}

Also include 'Bearer ' before the token on the front end.